build-tools-internal/src/main/groovy/elasticsearch.authenticated-testclusters.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
+
+def clusterCredentials = [
+        username: providers.systemProperty('tests.rest.cluster.username')
+                .forUseAtConfigurationTime()
+                .getOrElse('test_admin'),
+        password: providers.systemProperty('tests.rest.cluster.password')
+                .forUseAtConfigurationTime()
+                .getOrElse('x-pack-test-password')
+]
+
+tasks.withType(StandaloneRestIntegTestTask).configureEach {
+    systemProperty 'tests.rest.cluster.username', clusterCredentials.username
+    systemProperty 'tests.rest.cluster.password', clusterCredentials.password
+}
+
+testClusters.all {
+    testDistribution = 'DEFAULT'
```

Comment:
We should probably remove this bit. First, we want to slowly get away from having to use the default distro and test against the integ-test distro, and second we are setting this in the individual build scripts anyhow.

Reply:
fixed